### PR TITLE
ENG-10024 fix export view recover test fix in valgrind.

### PR DESCRIPTION
**Pull request by JadenW**
Changes in voltdbipc:
enable getUSOForExportTable;
tweak buffer size calculation in tableStreamSerializeMore;

### CODE REVIEW

File: src/frontend/org/voltdb/jni/ExecutionEngineIPC.java [hunk 1]
```diff
@@ -1352,10 +1368,11 @@ public void exportAction(boolean syncAction,
             retval[1] = results.getLong();
 
         } catch (final IOException e) {
+            e.printStackTrace();
```
Comment (vtkstef) [inline src/frontend/org/voltdb/jni/ExecutionEngineIPC.java]: why a printStackTrace() here?

Comment (JadenW) [inline src/frontend/org/voltdb/jni/ExecutionEngineIPC.java]: Sorry. this is for debug only. I will remove it.

File: tests/frontend/org/voltdb/regressionsuites/EEProcess.java [hunk 2]
```diff
@@ -72,6 +72,7 @@ public int port() {
         args.add("--leak-check=full");
         args.add("--show-reachable=yes");
         args.add("--num-callers=32");
+        args.add("--track-origins=yes");
```
Comment (akhanzode) [inline tests/frontend/org/voltdb/regressionsuites/EEProcess.java]: Do you want to set this option? It makes it slowwww

Comment (JadenW) [inline tests/frontend/org/voltdb/regressionsuites/EEProcess.java]: This should also for debug only. I will remove it.

**Pull request merged**